runtime/bcutil/ROMClassWriter.cpp

```diff
@@ -508,6 +508,12 @@ class ROMClassWriter::ConstantPoolWriter : public ConstantPoolMap::ConstantPoolV
 		_cursor->writeU32((cfrKind << BCT_J9DescriptionCpTypeShift) | BCT_J9DescriptionCpTypeMethodHandle, Cursor::GENERIC);
 	}
 
+	void visitConstantDynamic(U_16 cfrCPIndex, U_16 bsmIndex)
+	{
```

Comment:
Can you add an `/* assumes format: { ......} */` comment to this to match all the other visit* methods?

Reply:
added